.github/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: Colabot PY CI
+on:
+  pull_request:
+    branches: [main]
```

Comment:
this repo hasn't been converted from master -> main yet, so the workflow won't execute

Reply:
changed branch to master